### ISSUE TITLE
TypeError: that.views[view].widgets[wid] is undefined

### ISSUE THREAD

**Issue by hgschwibbe** [ioBroker/ioBroker.vis#656]
I created some template component for a socket to copy.

After I copied and configured my template 11 times I faced the following error:

![grafik](https://user-images.githubusercontent.com/1725648/221323826-dde57eed-2cda-41e7-b70a-dca3f142ebaa.png)
![grafik](https://user-images.githubusercontent.com/1725648/221322879-9a80d019-ad05-42e6-8764-15153d7c7c2b.png)

**Comment by hgschwibbe** [comment]
Before I forget, here is my anonymized project if you try to reproduce it:
[2023-02-25-main.zip](https://github.com/ioBroker/ioBroker.vis/files/10830763/2023-02-25-main.zip)

**Comment by Feuer-sturm** [comment]
@hgschwibbe please post as well the setting of your Web Adapter

**Comment by hgschwibbe** [comment]
![grafik](https://user-images.githubusercontent.com/1725648/221356087-60eb6dfa-afc9-4e60-9af8-6bd534cf9aad.png)

**Comment by hgschwibbe** [comment]
![grafik](https://user-images.githubusercontent.com/1725648/221356134-329f044c-e073-4b8d-b3d6-e35630f7d6f7.png)

**Comment by Feuer-sturm** [comment]
@hgschwibbe Does the behavior of your issue change if you activate "Reine Web-Sockets (iobroker.ws) verwenden"?

**Comment by hgschwibbe** [comment]
Yes it change. I can create more elements now but the console is full of error logs:
![grafik](https://user-images.githubusercontent.com/1725648/221376790-d152d77f-c436-4761-95a5-b2753892834c.png)

**Comment by hgschwibbe** [comment]
I also recognized that I am able to create more widgets when I reduce the amount of sensors.
I removed the Ampere sensor and the additional lightning indicator from the socket image to release more widgets.
It looks like the webapp scales down as the number of sensors increases.
